src/pagination/pagination.ts

```diff
@@ -191,6 +191,13 @@ export class NgbPagination implements OnChanges {
   @ContentChild(NgbPaginationNumber, {static: false}) tplNumber: NgbPaginationNumber;
   @ContentChild(NgbPaginationPrevious, {static: false}) tplPrevious: NgbPaginationPrevious;
 
+  /**
+   * The pagination alignment.
+   *
+   * Any of the Bootstrap flex utilties (i.e. "justify-content-center").
+   */
+  @Input() align: string;
```

Comment:
IMO you should have done either:
- `@Input() classNames: string` or
- `@Input() align: 'start' | 'center' | 'end' | etc... '`

Reply:
I didn't call it classNames since I didn't think that was clear where the class names were going, and there was already a size attribute it seemed like it would make sense to call it align. I wasn't going to enumerate all the values because there are much more than are used for the size parameter, and unless you have ivy and strict type checking enabled you're not going to know you're passing an "invalid" type (I haven't seen the warning myself, and I'm in the processes of testing ivy with our application so I've not had a chance to ratchet up the type checking)